crates/holochain/src/conductor/api/api_external/app_interface.rs

```diff
@@ -79,6 +82,59 @@ impl AppInterfaceApi {
                     .get_app_info(&installed_app_id)
                     .await?,
             )),
+            AppRequest::AgentInfo { dna_hash } => {
+                // Get app info to know which DNAs belong to this app
+                let app_info = self
```

Comment:
The implementation of these calls should be put in `holochain::conductor::conductor`, and here the conductor method is just called from there.

Reply:
done

Reply:
done